jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ssl/SslConnection.java

```diff
@@ -327,12 +330,16 @@ private int getBufferSize(ToIntFunction<SSLSession> bufferSizeFn)
 
     private void acquireEncryptedInput()
     {
+        if (!_lock.isHeldByCurrentThread())
+            throw new IllegalStateException();
```

Comment:
Our convention elsewhere is to name such methods with a `Locked` suffix.  These are private, so you can rename